torchao/quantization/marlin_qqq/README.md

```diff
@@ -0,0 +1,13 @@
+# Marlin QQQ
+
+Marlin QQQ kernel and Marlin kernel mainly have the following differences:
```

Comment:
Also can you add some description on the expected Sm support required, from the ifdefs looks like its sm80 +?

Reply:
finished